src/main/java/hw_8/SessionCloser.java

```diff
@@ -0,0 +1,7 @@
+package hw_8;
+
+import java.util.List;
+
+public interface SessionCloser {
```

Comment:
Add @FunctionalInterface annotation and read about that. If I'm not mistaken we considered this annotation on our previous lection

Reply:
fixed